icu4c/source/data/misc/likelySubtags.txt

```diff
@@ -65,6 +65,7 @@ likelySubtags:table(nofallback){
     arn{"arn_Latn_CL"}
     aro{"aro_Latn_BO"}
     arq{"arq_Arab_DZ"}
+    ars{"ars_Arab_SA"}
```

Comment:
The likelySubtags data is auto-generated in CLDR based on the <territoryInfo> data in supplementalData. Please file a CLDR issue to ensure that "ars" data gets added for "SA", otherwise this change will get wiped out with the next CLDR update.

Reply:
Already done: https://unicode-org.atlassian.net/browse/CLDR-13071

Reply:
Thanks